src/clj/nnts2/db/organization.clj

```diff
@@ -18,3 +18,29 @@
                                  sql/format)
                    {:identifiers utils/snake->kebab})
        first)))
+
```

Comment:
add-user and get-by-user both are interacting with a different resource than create organization.
You should separate them as you have separated member-spec and organization-spec